pandas/tests/resample/test_period_index.py

```diff
@@ -877,3 +891,53 @@ def test_sum_min_count(self):
             [3.0, np.nan], index=PeriodIndex(["2018Q1", "2018Q2"], freq="Q-DEC")
         )
         tm.assert_series_equal(result, expected)
+
+
+def test_period_index_frequency_ME_error_message():
+    msg = "Invalid frequency: 2ME"
+
+    with pytest.raises(ValueError, match=msg):
+        PeriodIndex(["2020-01-01", "2020-01-02"], freq="2ME")
+
+
+def test_resample_frequency_ME_error_message(series_and_frame):
+    msg = "Invalid frequency: 2ME"
+
+    obj = series_and_frame
+    with pytest.raises(ValueError, match=msg):
+        obj.resample("2ME")
+
+
+def test_period_range_frequency_ME_error_message():
+    msg = "Invalid frequency: 2ME"
+    with pytest.raises(ValueError, match=msg):
+        period_range("Jan-2000", "Dec-2000", freq="2ME")
```

Comment:
this one doesn't do any resampling? not sure it belongs in the `tests/resample` tests